dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -35,6 +35,9 @@ social.github.conf.desc=The configuration of the social login for GitHub.
 social.linkedin.conf=Linkedin Social Login
 social.linkedin.conf.desc=The configuration of the social login for Linkedin.
 
+social.openshift.conf=OpenShift Social Login
+social.openshift.conf.desc=The configuration of the social login for OpenShift.
```

Comment:
Change
"The configuration of the social login for OpenShift."
to
"The configuration of the social login for OpenShift server."